acceptance/Gemfile.lock

```diff
@@ -0,0 +1,214 @@
+GIT
```

Comment:
I think this `Gemfile.lock` is fine because it is under `acceptance/`. This makes sure that we run the acceptance suite always with the same set of gems.